torch/tensor.py

```diff
@@ -403,7 +403,7 @@ def __rsub__(self, other):
         return _C._VariableFunctions.rsub(self, other)
 
     def __rdiv__(self, other):
-        if self.dtype.is_floating_point:
+        if self.dtype.is_floating_point or self.dtype.is_complex:
```

Comment:
@anjali411 I guess there should be a test to verify #36671

Reply:
hey @kshitij12345 I think the existing test in test_torch for rdiv is sufficient to check the correct behavior of rdiv

Reply:
@anjali I am not sure if that is the case. Can you check the current test by undoing your fix to verify it fails. Thanks.